README.md

```diff
@@ -26,15 +27,15 @@ One button below has an `ng-click` attribute; the other has `data-ng-click` inst
 ```
 
 ```text
-Your answer goes here...
+no difference except that ng-click is not html-validator compliant.  For all intents and purposes they are the same
 ```
 
 ### Question 3
 
 Which of the three following options demonstrates the best usage of `ng-app`? **Explain your answer.**
 
 ```text
-Your answer goes here...
+A - data-ng-app designates the entry point of the application, so we would want to place it where there is the widest domain for directive, which is the entire html tag.
 ```
 
```

Comment:
There will often be multiple ng-app on a page. Your answer is sensible, but perhaps more often than not, there would be multiple ng-apps on the page. Binding the app to a div might make more sense, given that.